.github/workflows/deploy.yml

```diff
@@ -69,13 +72,15 @@ jobs:
           cylc_flow: ${{ github.event.inputs.cylc-flow-tag }}
           meto_rose: ${{ github.event.inputs.metomi-rose-tag }}
           cylc_rose: ${{ github.event.inputs.cylc-rose-tag }}
+          cylc_uis: ${{ github.event.inputs.cylc-uis-tag }}
         run: |
           # NOTE: Install with [all] so we can import plugins which may
           #       have extra dependencies.
           pip install  \
             "cylc-flow[all] @ git+https://github.com/cylc/cylc-flow@${cylc_flow}" \
             "metomi-rose[all] @ git+https://github.com/metomi/rose@${meto_rose}" \
-            "cylc-rose[all] @ git+https://github.com/cylc/cylc-rose@${cylc_rose}"
+            "cylc-rose[all] @ git+https://github.com/cylc/cylc-rose@${cylc_rose}" \
```

Comment:
I tried to test this, and get a dependency error:
```
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
    cylc-flow[all] 8.0rc1.dev0 depends on cylc-flow 8.0rc1.dev0 (from git+https://github.com/cylc/cylc-flow@master)
    cylc-rose[all] 0.3.0 depends on cylc-flow==8.0b3.*
```

would it be reasonable to loosen the `cylc-rose` dependency at https://github.com/cylc/cylc-rose/blob/c9eef34a118913a0137ac08d5b4330ac38454d49/setup.py#L31

Reply:
No, this version is correct.